.eslintrc.js

```diff
@@ -36,22 +36,13 @@ module.exports = {
             "error",
             2
         ],
-        "linebreak-style": [
+        "no-console": [
             "error",
-            "windows"
-        ],
-        "quotes": [
-            "warn",
-            "single"
-        ],
-        "no-unused-vars": [
-            "warn",
-            "all"
-        ],
-        "no-constant-condition": [
-            "warn"
-        ],
-        "no-console": "off"
+                 {"allow":
+                        ["warn", "info", "error"]
```

Comment:
Let's only allow `error` as it's the only one we are using (or does linting fail if you only leave `error`?)

Reply:
It works - I just thought that, probably, these ones will be needed too later. :)

Reply:
I want to avoid using the console as it's not great from a user's point of view (as it's not easily visible to them).